clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -174,4 +194,22 @@ public String toString() {
                 ')';
         }
     }
+
+    public static class Node {
+        private int nodeId;
```

Comment:
these need to be final. Also, we should understand what the result is for pre-KIP-853 nodes. Do they get an empty list of endpoints? That seems simple, but it needs to be spelled out.

Reply:
> Do they get an empty list of endpoints?

Yes.

> That seems simple, but it needs to be spelled out.

Do we need to write it down somewhere?